manifest.toml

```
@@ -0,0 +1,57 @@
+packaging_format = 2
+
+id = "paheko"
+name = "Paheko"
+description.en = "Software to manage association"
+description.fr = "Logiciel libre de gestion d'association"
+
+version = "1.3.0~ynh2"
+
+maintainers = ["rodinux"]
+
+[upstream]
+license = "GPL-3.0-or-later"
+website = "https://paheko.cloud"
+demo = "https://paheko.cloud/essai/"
+admindoc = "https://fossil.kd2.org/paheko/wiki?name=Documentation"
+code = "https://fossil.kd2.org/paheko/dir?ci=tip"
+
+[integration]
+yunohost = ">= 11.1.21"
```

```suggestion
yunohost = ">= 11.2"
```

cf proposed changes in install and upgrade for fpm config

---

ok